File: PHP/Events/AvailablePerformanceInfo.php
<?php
namespace LTD_API;

class AvailablePerformanceInfo
{
    private $performanceId;
    //date format
    private $performanceDate;
    private $minimumTicketPrice;
    //array of AvailableTicketInfo
    private $availableTicketInfos;
    //array of AvailableTicketsBlockInfo
    private $availableTicketsBlockInfos;
    
    public function getPerformanceId()
    {
        return $this->performanceId;
    }
    public function setPerformanceId($performanceId)
    {
        $this->performanceId = $performanceId;
    }

    public function getPerformanceDate()
    {
        return $this->performanceDate;
    }
    public function setPerformanceDate($performanceDate)
    {
        $this->performanceDate = $performanceDate;
    }

    public function getMinimumTicketPrice()
    {
        return $this->minimumTicketPrice;
    }
    public function setMinimumTicketPrice($minimumTicketPrice)
    {
        $this->minimumTicketPrice = $minimumTicketPrice;
    }

    public function getAvailableTicketInfos()
    {
        return $this->availableTicketInfos;
    }
    public function setAvailableTicketInfos($availableTicketInfos)
    {
        $this->availableTicketInfos = $availableTicketInfos;
    }

    public function getAvailableTicketsBlockInfos()
    {
        return $this->availableTicketsBlockInfos;
    }
    public function setAvailableTicketsBlockInfos($availableTicketsBlockInfos)
    {
        $this->availableTicketsBlockInfos = $availableTicketsBlockInfos;
    }
}

File: Java/Events/AvailablePerformanceInfo.java
package LTD_API;

import java.util.Date;
import java.util.List;

public class AvailablePerformanceInfo
{
    private int performanceId;
    private Date performanceDate;
    private decimal minimumTicketPrice;
    private List<AvailableTicketInfo> availableTicketInfos;
    private List<AvailableTicketsBlockInfo> availableTicketsBlockInfos;
    
    public int getPerformanceId()
    {
        return performanceId;
    }
    public void setPerformanceId(int performanceId)
    {
        this.performanceId = performanceId;
    }

    public Date getPerformanceDate()
    {
        return performanceDate;
    }
    public void setPerformanceDate(Date performanceDate)
    {
        this.performanceDate = performanceDate;
    }

    public decimal getMinimumTicketPrice()
    {
        return minimumTicketPrice;
    }
    public void setMinimumTicketPrice(decimal minimumTicketPrice)
    {
        this.minimumTicketPrice = minimumTicketPrice;
    }

    public List<AvailableTicketInfo> getAvailableTicketInfos()
    {
        return availableTicketInfos;
    }
    public void setAvailableTicketInfos(List<AvailableTicketInfo> availableTicketInfos)
    {
        this.availableTicketInfos = availableTicketInfos;
    }

    public List<AvailableTicketsBlockInfo> getAvailableTicketsBlockInfos()
    {
        return availableTicketsBlockInfos;
    }
    public void setAvailableTicketsBlockInfos(List<AvailableTicketsBlockInfo> availableTicketsBlockInfos)
    {
        this.availableTicketsBlockInfos = availableTicketsBlockInfos;
    }
}

